### PR TITLE
Show cancel button

### PR DESCRIPTION
**Pull request by johnbyrne727rit**
For this Trello card:
https://trello.com/c/1JxXeFLJ/78-no-cancel-button-in-admin-create-new-show-page

Adds cancel button to the admin create show page:
cancel-button

### CODE REVIEW

File: frontend/src/Admin/components/CreateShowForm.js [hunk 1]
```diff
@@ -227,16 +227,34 @@ class CreateShowForm extends Component {
...
+                  disabled={isSubmitting}
+                >
+                  Create
+                </Button>
+              </Col>
+              <Col>
+                <Button
+                  type='button'
+                  color='danger'
+                  className="float-right"
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/components/CreateShowForm.js:247]: Slides it to the right of it's column.

File: frontend/src/Admin/components/CreateShowForm.js [hunk 2]
```diff
@@ -227,16 +227,34 @@ class CreateShowForm extends Component {
...
+                  style={{
+                    cursor: 'pointer'
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/components/CreateShowForm.js:249]: This is just to match the other button's style.

File: frontend/src/Admin/components/CreateShowForm.js [hunk 3]
```diff
@@ -227,16 +227,34 @@ class CreateShowForm extends Component {
...
+                  }}
+                  disabled={isSubmitting}
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/components/CreateShowForm.js:251]: Should definitely not be able to hit cancel after hitting submit.

File: frontend/src/Admin/components/CreateShowForm.js [hunk 4]
```diff
@@ -227,16 +227,34 @@ class CreateShowForm extends Component {
...
-              type='submit'
-              color='primary'
-              style={{
-                cursor: 'pointer'
-              }}
-              disabled={isSubmitting}
-            >
-              Create
-            </Button>
+            <Row>
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/components/CreateShowForm.js:230]: Without the `Row` and `Col` bootstrap, telling the cancel button to float right put it in the middle of the screen.

File: frontend/src/Admin/components/CreateShowForm.js [hunk 5]
```diff
@@ -227,16 +227,34 @@ class CreateShowForm extends Component {
...
+              <Col>
+                <Button
+                  type='button'
+                  color='danger'
+                  className="float-right"
+                  style={{
+                    cursor: 'pointer'
+                  }}
+                  disabled={isSubmitting}
+                  onClick={done}
```
Comment (johnbyrne727rit) [inline frontend/src/Admin/components/CreateShowForm.js:252]: `done` just dispatches a url change to redux, so it works as cancel too.

**Review by JohnPMurray — APPROVED** [review]
looks good to me

**Review by moiseslorap — APPROVED** [review]

**Pull request merged**